src/app/routes.ts

```diff
@@ -81,3 +81,5 @@ export const authenticatedRoutes = {
         path: '*',
     },
 };
+
+export const REDIRECT_TO_PARAM_NAME = 'redirect_to';
```

Comment:
Weird request - but can you toss this to a top. It feels like it would be easy to miss known search params listing at the bottom of a routes file.

Reply:
done!